Add P-R lettered article stubs from glossary (#3867)
ppy/osu-wiki: wiki/Ranking/en.md

---
stub: true
tags:
  - pp
  - leaderboards
  - ranking
---

# Ranking

**Ranking** (alternatively called "leaderboards") is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.

> All but the individual beatmap rankings can be navigated to by hovering the `rankings` menu on the website header. For the individual beatmap ranking, they can be found on their beatmap info page.

Reviewer: ```suggestion
All but the individual beatmap rankings can be navigated to by hovering over the `rankings` menu on the website header. For the individual beatmap ranking, they can be found on their beatmap info page.
```

## Performance points ranking

> *Main page: [Performance points](/wiki/Performance_points)*

Reviewer: ```suggestion
*See also: [Performance points](/wiki/Performance_points)*
```

The performance points ranking describes a player's skill-level in comparison to other players. This leaderboard is determined by the total number of performance points an individual player has, and can be seen on a player's profile and on the [performance leaderboards](https://osu.ppy.sh/rankings/osu/performance).

## Score ranking

The score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance points ranking.

> ## kudosu! ranking

Reviewer: ```suggestion
## Kudosu ranking
```

Reviewer 2: @Joehuu suggested to use `kudosu!` (stylized like that) in https://github.com/ppy/osu-wiki/pull/3867#discussion_r527128471, I'm also leaning that way too.... I was about to add it as a term in the ASC to be spelt like that.

> The kudosu! ranking compares users' accumulated [kudosu!](/wiki/Modding/Kudosu) and lists them accordingly. This ranking requires no playing skill as kudosu! is gained as part of the [modding process](/wiki/Modding).

Reviewer: ```suggestion
The Kudosu ranking compares users' accumulated [Kudosu](/wiki/Modding/Kudosu) and lists them accordingly. This ranking requires no playing skills, as Kudosu is gained as a part of the [modding process](/wiki/Modding).
```

Reviewer 2: Will apply some of those changes
```suggestion
The kudosu! ranking compares users' accumulated [kudosu!](/wiki/Modding/Kudosu) and lists them accordingly. This ranking requires no playing skills, as Kudosu is gained as a part of the [modding process](/wiki/Modding).
```

---
stub: true
tags:
  - pp
  - leaderboards
  - ranking
---

# Ranking

> **Ranking**, alternatively called *leaderboards*, is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.

Reviewer 3: ```suggestion
**Ranking** (alternatively called "leaderboards") is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.
```

osu! currently has five types of rankings:

> - Performance point ranking: based on [performance points](/wiki/Performance_Points).

Reviewer 3: ```suggestion
- Performance points ranking: based on [performance points](/wiki/Performance_Points).
```

- Score ranking: based on total ranked score.
- Kudosu! ranking: based on [kudosu!](/wiki/Modding/Kudosu) count.
- Spotlights rankings: based on [score](/wiki/Score) on the currently spotlighted beatmaps.
- Beatmap rankings: based on score for individual [beatmaps](/wiki/Beatmaps).
- Country rankings: based on performance points of players from a country.

> The performance point ranking, score ranking, and Kudosu! ranking can be navigated to by using the `Rankings` dropdown panel at the top of the webpage and choosing the `Performance` option.

Reviewer 3: ```suggestion
The performance points ranking, score ranking, and Kudosu! ranking can be navigated to by using the `Rankings` dropdown panel at the top of the webpage and choosing the `Performance` option.
```

> ## Performance point ranking
> 
> Performance point ranking describes a player's skill-level in comparison to other players. This leaderboard is determined by performance points, and can be seen on a player's profile and on the [performance leaderboards](https://osu.ppy.sh/rankings/osu/performance).

Reviewer 3: ```suggestion
## Performance points ranking

*Full explanation: [Performance Points](/wiki/Performance_Points)*

The performance points ranking describes a player's skill-level in comparison to other players. This leaderboard is determined by the total number of performance points an individual player has, and can be seen on a player's profile and on the [performance leaderboards](https://osu.ppy.sh/rankings/osu/performance).
```

## Score ranking

> Score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance point ranking.

Reviewer 3: ```suggestion
Score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance points ranking.
```

## Kudosu! ranking

Kudosu! ranking compares users' accumulated Kudosu and lists them accordingly. This ranking requires no playing skill as Kudosu! is gained as part of the [modding process](/wiki/Modding).

## Spotlights rankings

> Spotlights rankings are time-limited leaderboards that compare players' skill on a specific set of beatmaps each spotlight season. Players who rank highly on spotlight rankings receive rewards.

Reviewer 3: ```suggestion
Spotlights rankings are time-limited leaderboards that compare players' skill on a specific set of beatmaps each spotlight season. Players who rank highly on spotlight rankings will receive specific awards as a result of there hard work.
```

## Beatmap rankings

> Beatmap rankings compare players' scores against eachother on individual beatmaps. They appear once a beatmap has been Qualified, Ranked, Approved or Loved.  First place rankings on these leaderboards can be seen on a user's profile page, and comes with a bit of prestige.

Reviewer 3: ```suggestion
The beatmap rankings compare players' scores against each other on individual beatmaps. They appear once a beatmap has been [Qualified](/wiki/Beatmaps#qualified), [Ranked](/wiki/Beatmaps#ranked), [Approved](/wiki/Beatmaps#approved) or Loved. First place rankings on these leaderboards can be seen on a user's profile page, and comes with a bit of prestige.
```

osu! currently has five types of rankings:

- Performance points ranking: based on [performance points](/wiki/Performance_points).
- Score ranking: based on total ranked score.
- Kudosu! ranking: based on [kudosu!](/wiki/Modding/Kudosu) count.
- Spotlights rankings: based on [score](/wiki/Score) on the currently spotlighted beatmaps.
- Beatmap rankings: based on score for individual [beatmaps](/wiki/Beatmaps).
- Country rankings: based on performance points of players from a country.

The performance points ranking, score ranking, and Kudosu! ranking can be navigated to by using the `Rankings` dropdown panel at the top of the webpage and choosing the `Performance` option.

## Performance points ranking

*Main page: [Performance points](/wiki/Performance_points)*

The performance points ranking describes a player's skill-level in comparison to other players. This leaderboard is determined by the total number of performance points an individual player has, and can be seen on a player's profile and on the [performance leaderboards](https://osu.ppy.sh/rankings/osu/performance).

## Score ranking

> Score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance points ranking.

Reviewer 4: ```suggestion
The score ranking compares players' total score and lists them accordingly. Ranking highly on this leaderboard requires a large time commitment, but requires relatively less mechanical skill than the performance points ranking.
```

> ## Kudosu! ranking

Reviewer 4: Hmm, I'm not even sure what is the official way to format this. See https://osu.ppy.sh/help/wiki/Modding/Kudosu and the profile for example. By the way, I'm leaning towards the `kudosu!` format.

```suggestion
## kudosu! ranking
```

Reviewer 2: ok, I will note this as well

> Kudosu! ranking compares users' accumulated Kudosu and lists them accordingly. This ranking requires no playing skill as Kudosu! is gained as part of the [modding process](/wiki/Modding).

Reviewer 4: ```suggestion
The kudosu! ranking compares users' accumulated [kudosu!](/wiki/Modding/Kudosu) and lists them accordingly. This ranking requires no playing skill as kudosu! is gained as part of the [modding process](/wiki/Modding).
```

## Spotlights rankings

> Spotlights rankings are time-limited leaderboards that compare players' skill on a specific set of beatmaps each spotlight season. Players who rank highly on spotlight rankings will receive specific awards as a result of their hard work.

Reviewer 4: Spotlights is proper.

```suggestion
The Spotlights rankings are time-limited leaderboards that compare players' skill on a specific set of beatmaps each [Spotlights](/wiki/Beatmap_Spotlights) season. Players who rank highly on Spotlights rankings will receive specific awards as a result of their hard work.
```
